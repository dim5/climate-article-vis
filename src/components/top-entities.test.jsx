import { render } from '@testing-library/react';

import TopEntities from './top-entities';

it('should render', () => {
  const { container } = render(<TopEntities />);
  expect(container.innerHTML).toMatchSnapshot();
});
