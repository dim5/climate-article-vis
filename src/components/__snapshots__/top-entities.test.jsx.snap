// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 1`] = `"<div><div class=\\"top-entities__PercentItem-z8u710-0 dWKeOt\\">UNITED STATES <span class=\\"top-entities___StyledSpan-z8u710-1 gWCfPW\\">4705</span></div><div class=\\"top-entities__PercentItem-z8u710-0 fqoZQB\\">EARTH <span class=\\"top-entities___StyledSpan-z8u710-1 gWCfPW\\">2308</span></div><div class=\\"top-entities__PercentItem-z8u710-0 cuFhKZ\\">AMERICA <span class=\\"top-entities___StyledSpan-z8u710-1 gWCfPW\\">1995</span></div><div class=\\"top-entities__PercentItem-z8u710-0 bdTIxN\\">AUSTRALIA <span class=\\"top-entities___StyledSpan-z8u710-1 gWCfPW\\">1701</span></div><div class=\\"top-entities__PercentItem-z8u710-0 fHVtVx\\">DONALD TRUMP <span class=\\"top-entities___StyledSpan-z8u710-1 gWCfPW\\">1682</span></div><div class=\\"top-entities__PercentItem-z8u710-0 gjOtGd\\">UNITED NATIONS <span class=\\"top-entities___StyledSpan-z8u710-1 gWCfPW\\">1673</span></div><div class=\\"top-entities__PercentItem-z8u710-0 kmwmfp\\">UNITED KINGDOM <span class=\\"top-entities___StyledSpan-z8u710-1 gWCfPW\\">1441</span></div><div class=\\"top-entities__PercentItem-z8u710-0 IYJbp\\">CALIFORNIA <span class=\\"top-entities___StyledSpan-z8u710-1 gWCfPW\\">1429</span></div><div class=\\"top-entities__PercentItem-z8u710-0 fHZtIB\\">CHINA <span class=\\"top-entities___StyledSpan-z8u710-1 gWCfPW\\">1422</span></div><div class=\\"top-entities__PercentItem-z8u710-0 dtsmNN\\">PARIS AGREEMENT <span class=\\"top-entities___StyledSpan-z8u710-1 gWCfPW\\">1419</span></div><div class=\\"top-entities__PercentItem-z8u710-0 iFpqiZ\\">EUROPE <span class=\\"top-entities___StyledSpan-z8u710-1 gWCfPW\\">1389</span></div><div class=\\"top-entities__PercentItem-z8u710-0 kUBaxN\\">CO2 <span class=\\"top-entities___StyledSpan-z8u710-1 gWCfPW\\">1011</span></div><div class=\\"top-entities__PercentItem-z8u710-0 iTMixN\\">INDIA <span class=\\"top-entities___StyledSpan-z8u710-1 gWCfPW\\">968</span></div><div class=\\"top-entities__PercentItem-z8u710-0 jJzPet\\">ARCTIC <span class=\\"top-entities___StyledSpan-z8u710-1 gWCfPW\\">905</span></div><div class=\\"top-entities__PercentItem-z8u710-0 fFNnBh\\">EU <span class=\\"top-entities___StyledSpan-z8u710-1 gWCfPW\\">892</span></div></div>"`;
